lib/cdo/http_cache.rb

```diff
@@ -230,12 +230,21 @@ def self.config(env)
             query: false,
             headers: ALLOWLISTED_HEADERS,
             cookies: default_cookies
+          },
+          # NextJS assets path for the marketing app
+          {
+            path: '/_next/*',
+            proxy: 'marketing',
+            headers: ALLOWLISTED_HEADERS,
+            cookies: default_cookies,
+            include_marketing_router_lambda: true,
           }
         ],
         # Remaining Pegasus paths are cached, and vary only on language, country, and default cookies.
         default: {
           headers: LANGUAGE_HEADER + COUNTRY_HEADER,
```

Comment:
Do we need to append Headers and Cookies used by NextJS such as the ones below and will any of those headers/cookies have a range of values that varies beyond a small enumerated set? If any of these cookies or headers will have a value that is a unique id or token, then CloudFront would not be effective at caching


Headers:
- accept
- rsc
- next-router-prefetch
- next-router-state-tree
- next-url
- x-prerender-revalidate


Cookies:

- `__prerender_bypass` (For draft mode)
- `language_ge_region` (not needed at this time)

Reply:
No headers/cookies need to be appended at this time. The one cookie that has a unique value is `__prerender_bypass` but that is intentional to bypass caching.

Reply:
To further clarify, I tested the following configuration on the adhoc cloudfront manually and it appears to work as expected:

---

**Behavior: `/_next/*`**

* Cache Policy: Managed-CachingOptimized
* Request Policy: Not Set

This should work on `/_next/static*` as well because those files are hashed. We can apply the most aggressive cache policy on those. In fact, the default cache control set by Next is: `public, max-age=31536000, immutable` (one year) when run on the application.

---

**Behavior: `/api/*`**

* Cache Policy: Managed-CachingDisabled
* Request Policy: Managed-AllViewer

This would be expected as these are dynamic APIs and shouldn't be cached.

Reply:
> No headers/cookies need to be appended at this time. The one cookie that has a unique value is `__prerender_bypass` but that is intentional to bypass caching and would only be used by content editors while previewing content.

OK - at some point, we'll need to add the `__prerender_bypass` cookie to the cache key for the Default Behavior (and any other Behavior that might serve dynamically generated NextJS content such as `/api/*`).

Reply:
The `__prerender_bypasss` cookie is only used by content editors directly (meaning via `code.marketing-sites.code.org`). This cookie is in the cache key as part of the PR i merged yesterday, so I think this is covered. The pegasus cloudfront doesn't need this cookie because this cookie is only used via the direct cloudfront.